readme.md

```diff
@@ -805,6 +805,40 @@ echo $faker->firstKanaName; // "ハルカ"
 echo $faker->lastKanaName; // "ナカジマ"
 ```
 
+### `Faker\Provider\kk_KZ\Company`
```

Comment:
No need to document the formatted that exist in every language, like `name` or `company`. Please document here only the formatters specific to your locale.